crowbar_framework/app/models/api/node.rb

```diff
@@ -15,6 +15,56 @@
 #
 
 module Api
-  class Node
+  class Node < Tableless
+    def repocheck(options = {})
+      addon = options.fetch(:addon, "os")
+      features = ["os"]
+      features.push(addon) if addon
+
+      {}.tap do |ret|
```

Comment:
`ret` means `return` in short, I just used it because we are using it in many places in crowbar, not sure what else to call it